admin_manual/enterprise/installation/oracle_db_configuration.rst

```diff
@@ -8,108 +8,72 @@ Oracle Database Setup
-ownCloud with all of the prerequisites.
+The document assumes that you already have your Oracle instance running, and have provisioned the needed resources. It also assumes that you have installed ownCloud with all of the prerequisites.
 
 Configuring Oracle
 ==================
 
 Setting up the User Space for ownCloud
 --------------------------------------
 
-Step one, if it has not already been completed by your :abbr:`DBA (DataBase Administrator)`, provision a user
-space on the Oracle instance for ownCloud.  This can be done by logging in as a
-DBA and running the script below:
+Step one, if it has not already been completed by your :abbr:`DBA (DataBase Administrator)`, provision a user space on the Oracle instance for ownCloud.  
+This can be done by logging in as a DBA and running the script below:
 
 ::
 
   CREATE USER owncloud IDENTIFIED BY password;
   ALTER USER owncloud DEFAULT TABLESPACE users TEMPORARY TABLESPACE temp QUOTA unlimited ON users;
   GRANT create session, create table, create procedure, create sequence, create trigger, create view, create synonym, alter session TO owncloud;
 
-Substitute an actual password for ``password``.  Items like TableSpace, Quota etc. will be determined by your DBA.
+Substitute an actual password for ``password``.  
+Items like *TableSpace*, *Quota* etc., will be determined by your DBA (database administrator).
 
-Downloading and Installing the Oracle Instant Client
-----------------------------------------------------
+Add OCI8 Client Support
+-----------------------
 
-As our example system is Oracle Enterprise Linux, it is
-necessary to go to the Oracle site and download the `Oracle Instant Client`_ for
-your OS Distribution.
+To install `the OCI8 extension`, you first need to download ``oracle-instantclient12.2-devel-12.2.0.1.0-1.x86_64.rpm`` and ``oracle-instantclient12.2-basic-12.2.0.1.0-1.x86_64.rpm`` `from Oracle`_, and install them using the following command:
 
-.. _Oracle Instant Client: http://www.oracle.com/technetwork/database/features/instant-client/index-097480.html
+.. code-block:: console
 
-.. note:: Download the instant client and the instant client SDK and place them
-   in a directory on the server, in this example they are RPM packages.
+   rpm --install oracle-instantclient12.2-basic-12.2.0.1.0-1.x86_64.rpm \
+     oracle-instantclient12.2-devel-12.2.0.1.0-1.x86_64.rpm
 
-* Install the basic client from the RPM.  Use the ``rpm –ivh`` command
-* Install the SDK RPM package.  Use the ``rpm –ivh`` command
+With that done, you're now ready to install the OCI8 extension. 
+To do that, first add a configuration file for it by using the command below. 
 
-At this point, the Oracle Instant client is installed on the ownCloud Host (in
-the home directory).
+.. code-block:: console
+   
+   cat << EOF > /etc/opt/rh/rh-php70/php.d/20-oci8.ini
+   ; Oracle Instant Client Shared Object extension
+   extension=oci8.so
```

Comment:
no idea on what this depends on - I just realized that this is the case sometimes 😕